### ISSUE TITLE
add useful loss functions

### ISSUE THREAD

**Issue by JorisRoels** [saeyslab/neuralnets#2]
Currently implemented:
- Cross entropy
- Lp-type losses such as MSE and MAD
- KLD

I think the following architectures would be useful to have:
- Dice / Jaccard

Feel free to add more!
